File: sunbin/day19/python/001_가위바위보.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120839

def solution(rsp):
    answer = ""
    for i in rsp:
      if i == "2":
        answer += "0"
      elif i == "0":
        answer += "5"
      else:
        answer += "2"
    
    return answer


# 다른풀이
def solution_b(rsp):
    d = {'0':'5','2':'0','5':'2'}
    return ''.join(d[i] for i in rsp)
    
solution("205")

File: sunbin/day19/001_가위바위보.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120839
// 테스트

function solution(rsp) {
  const rspArr = {
    2: 0,
    0: 5,
    5: 2
  }

  let answer = ''
  for (const i of rsp) {
    answer += String(rspArr[i])
  }

  // 더 간단히!
  // let answer = [...rsp].map(v => rspArr[v]).join("");

  return answer
}